File: sqlConnect.php
<?php
$servername = "localhost";
$username = "root";
$password = "password";
$dbname = "classroom";

//creating connection
$conn = new mysqli($servername, $username, $password, $dbname);

//Checking connection
if ($conn->connect_error) {
  die("Failed to Connect: " .$conn->connect_error);
}

//creating query
$sql = "SELECT name, age2, gradeLevel FROM students';
$result = $conn->query($sql);

//outing putting data for each row
if ($result-> num_rows > 0) {
  while ($row = $result->fetch_assoc()) {
    echo "Name: ".$row["name"]. " -Age: ". $row["age2"]. " - Grade Level ". $row["gradeLevel"]."\n".\n";
    }
}
else {
  echo "0 results";
}
//closeing connection
$conn->close();

 ?>
    

File: sqlConnect.py
import pymysql

#connecting to database
db = pymysql.connect("localhost", "root", "password", "classroom")

#preparing object with cursor()
mycursor = db.cursor(pymysql.cursors.DictCursor)

#show tables/data
sql1 = "SELECT * from students"

#executing SQL query with execute()
mycursor.execute(sql1)

data = cur.fetchall()
for row in data:
	print("Name: " + row['name'] + " Age: " + str(row['age2']) + " Grade level: " + str(row['gradeLevel'])) 

#server disconnecting
cur.close()
db.close()
